backend/main.py

```diff
@@ -123,32 +123,48 @@ async def websocket_endpoint(websocket: WebSocket):
             elif message["type"] == "bid":
                 if not game.ongoing:
                     await manager.send_personal_message_lobby(
-                        {"type": "error", "message": "Game has not started."}
+                        {"type": "error", "message": "Game has not started."}, websocket
                     )
                 else:
                     amount = message["amount"]
+                    game.bid(websocket, amount)
+                    bids_dict = {}
+                    for ws in game.bids:
+                        bids_dict[current_players[ws]] = game.bids[ws]
+                    cash_dict = {}
+                    for ws in game_handler.cash:
+                        cash_dict[current_players[ws]] = game_handler.cash[ws]
```

Comment:
```suggestion
                    # TODO: find a way to avoid recomputing the whole dicts each time a player makes a bid
                    
                    # Compute the mappings from player_name -> cash / bid amounts for the frontend
                    bids_dict = {}
                    for ws in game.bids:
                        bids_dict[current_players[ws]] = game.bids[ws]
                    cash_dict = {}
                    for ws in game_handler.cash:
                        cash_dict[current_players[ws]] = game_handler.cash[ws]
```
Le TODO c'est pour plus tard, it's good for now